doc/guide/Guide_Installation.ipynb

```diff
@@ -73,28 +77,33 @@
     "\n",
     "\n",
     "#### Workaround option B:\n",
-    "In a terminal window, enter:\n",
-    "\n",
+    "Start Anaconda, create a new python_env environment (just click create and enter the name, then press Create) and install latest Spyder in there (currently 4.2.5). Start this version of Spyder. After Spyder has started, navigate to *Preferences > Python Interpreter > Use the following interpreter* and paste the output of the following terminal command into the text box:\n",
     "```bash\n",
     "conda activate climada_env \n",
-    "conda install spyder-kernels \n",
     "python -c \"import sys; print(sys.executable)\" \n",
-    "--> returns a path, like /Users/XXX/opt/anaconda3/envs/climada_env/bin/python\n",
+    "# this returns a path, like /Users/XXX/opt/anaconda3/envs/climada_env/bin/python\n",
     "```\n",
+    "Now, obtain the version of the package `spyder-kernels` (currently 1.10.2) that has automatically been installed along with Spyder in the Anaconda environment and install the same version of `spyder-kernels` in the `climada_env` environment:\n",
```

Comment:
In a different thread on Slack we found that it's crucial to install a compatible version of `spyder-kernels` in your `climada_env`. If you don't specify a version, the latest develop version will be installed which is for Spyder 5 and not for the (stable) Spyder 4.